### PR TITLE
Create tag and show Question list

### PR DESCRIPTION
**Pull request by duykhanh21**

### CODE REVIEW

File: src/containers/CreateQuestion.jsx [hunk 1]
```diff
@@ -50,6 +50,11 @@ class CreateQuestion extends Component {
           handleChange={ this.handleFieldChange('content') }
         />
         <br />
+        Tags:
+        <InputField type={ 'text' } placeholder={' At least one, max 5 tags'}
```
Comment (yeuem1vannam) [inline src/containers/CreateQuestion.jsx]: `type="text"`

File: src/containers/Question.jsx [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
-    const questionId = this.props.params.questionId;
-    store.dispatch(this.getQuestionById(questionId));
+    console.log(this.props);
+    if (!this.props.status) {
+      const questionId = this.props.params.questionId;
+      store.dispatch(this.getQuestionById(questionId));
+    }
   }
 
   getQuestionById(id) {
```
Comment (yeuem1vannam) [inline src/containers/Question.jsx]: This method should be move to `question_actions`.

File: src/containers/VisibleQuestionList.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import Question from '../containers/Question';
+import { fetchQuestion } from '../actions/questions_action';
+import store from '../store';
+import lodash from 'lodash';
```
Comment (taduyanh) [inline src/containers/VisibleQuestionList.jsx:8]: did u include lodash to package.json ?

**Comment by taduyanh** [comment]
fix conflict @duykhanh21

File: src/containers/Question.jsx [hunk 4]
```diff
@@ -1,15 +1,18 @@
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 import C from '../constants';
 import Firebase from 'firebase';
 import store from '../store';
 import { connect } from 'react-redux';
-
+import isPlainObject from 'lodash/lang/isPlainObject';
```
Comment (taduyanh) [inline src/containers/Question.jsx]: do u really need to import specified isPlanObject function like this ?
is there any special reason.

In normal case we just need to import lodash and call `lodash.isPlainObject`

```
```

**Comment by yeuem1vannam** [comment]
:o:K
OMW

**Pull request merged**